docs/index.rst

```diff
@@ -101,7 +101,7 @@ The primary interaction with Paste Deploy is through its configuration
 files.  The primary thing you want to do with a configuration file is
 serve it.  To learn about serving configuration files, see `the
 ``paster serve`` command
-<http://pythonpaste.org/script/#paster-serve>`_.
+<https://web.archive.org/web/20160305001925/http://pythonpaste.org:80/script#paster-serve>`_.
```

Comment:
The correct (and new) link here is https://pastescript.readthedocs.io/en/latest/#paster-serve

Reply:
Ah, that makes sense. Your previous comment was on the incorrect line, which threw me. Fixed in 3c8dfafd4797ac889f233f45d45af8380f45d15f